main.go

```diff
@@ -146,6 +166,12 @@ func main() {
 	flags.Var(&options.collectors, "collectors", fmt.Sprintf("Comma-separated list of collectors to be enabled. Defaults to %q", &defaultCollectors))
 	flags.StringVar(&options.namespace, "namespace", metav1.NamespaceAll, "namespace to be enabled for collecting resources")
 
+	// Auth flags.
+	flags.BoolVar(&options.auth.Authentication.Anonymous.Enabled, "anonymous-auth", true, "Enables anonymous requests to the Kubelet server. Requests that are not rejected by another authentication method are treated as anonymous requests. Anonymous requests have a username of system:anonymous, and a group name of system:unauthenticated.")
```

Comment:
Maybe `Kubelet server` should be changed to `kube-state-metrics`.

Reply:
yes absolutely :slightly_smiling_face: